api/src/main/java/org/pmiops/workbench/cloudtasks/TaskQueueService.java

```diff
@@ -29,44 +28,58 @@
-  private static final String CREATE_WORKSPACE_QUEUE_NAME = "createWorkspaceQueue";
-  private static final String DUPLICATE_WORKSPACE_QUEUE_NAME = "duplicateWorkspaceQueue";
-  private static final String DELETE_TEST_WORKSPACES_QUEUE_NAME = "deleteTestUserWorkspacesQueue";
-  private static final String DELETE_RAWLS_TEST_WORKSPACES_QUEUE_NAME =
-      "deleteTestUserRawlsWorkspacesQueue";
-  private static final String FREE_TIER_BILLING_QUEUE = "freeTierBillingQueue";
-  private static final String INITIAL_CREDITS_EXHAUSTION_QUEUE = "initialCreditsExhaustionQueue";
-  private static final String CHECK_CREDITS_EXPIRATION_FOR_USER_IDS_QUEUE_NAME =
-      "checkCreditsExpirationForUserIDsQueue";
-  private static final String DELETE_WORKSPACE_ENVIRONMENTS_QUEUE_NAME =
-      "deleteUnsharedWorkspaceEnvironmentsQueue";
+
+  // when adding or updating a queue, it's important to also update:
+  // queue.yaml (the deployment process uses this file to actually create the queue)
+  // workbench-terraform-modules/modules/workbench/modules/monitoring/modules/alert_policies/assets/alert_policies
+  public static final TaskQueuePair ACCESS_EXPIRATION_EMAIL =
+      new TaskQueuePair("accessExpirationEmailQueue", "sendAccessExpirationEmails");
+  public static final TaskQueuePair AUDIT_PROJECTS =
+      new TaskQueuePair("auditProjectQueue", "auditProjectAccess");
+  public static final TaskQueuePair CREATE_WORKSPACE =
+      new TaskQueuePair("createWorkspaceQueue", "createWorkspace");
+  public static final TaskQueuePair DELETE_RAWLS_TEST_WORKSPACES =
+      new TaskQueuePair("deleteTestUserRawlsWorkspacesQueue", "deleteTestUserWorkspacesInRawls");
+  public static final TaskQueuePair DELETE_TEST_WORKSPACES =
+      new TaskQueuePair("deleteTestUserWorkspacesQueue", "deleteTestUserWorkspaces");
+  public static final TaskQueuePair DELETE_UNSHARED_WORKSPACE_ENVIRONMENTS =
+      new TaskQueuePair(
+          "deleteUnsharedWorkspaceEnvironmentsQueue", "deleteUnsharedWorkspaceEnvironments");
+  public static final TaskQueuePair DUPLICATE_WORKSPACE =
+      new TaskQueuePair("duplicateWorkspaceQueue", "duplicateWorkspace");
+  public static final TaskQueuePair EGRESS_EVENT =
+      new TaskQueuePair("egressEventQueue", "processEgressEvent");
+  public static final TaskQueuePair SYNCHRONIZE_ACCESS =
+      new TaskQueuePair("synchronizeAccessQueue", "synchronizeUserAccess");
+
+  // RDR exporting uniquely uses the same queue for two endpoints
+
+  public static final String RDR_EXPORT_QUEUE_NAME = "rdrExportQueue";
+  public static final TaskQueuePair RDR_RESEARCHER =
+      new TaskQueuePair(RDR_EXPORT_QUEUE_NAME, "exportResearcherData");
+  public static final TaskQueuePair RDR_WORKSPACE =
+      new TaskQueuePair(RDR_EXPORT_QUEUE_NAME, "exportWorkspaceData");
+
+  // initial credits queues and cloud tasks:
+  //
+  // INITIAL_CREDITS_EXPIRATION - run as part of the cron checkInitialCreditsExpiration to check
```

Comment:
please review this whole comment.  does it help explain the differences?

Reply:
> I think the comment is well written, but I am confused as to why exhaustion has subtasks. I don't think that that needs to be changed here, but I wonder if that process can be simplified.

Reply:
Yeah, I have been thinking about that.  My guess is that it was too memory- or time-intensive.